### PR TITLE
Create bug issue template

### PR DESCRIPTION
**Pull request by jhoule86**
- Bug report template
- The labels `new` and `bug` will be added each time someone uses this, so that we know to review it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[ie: New Warehouse mapping stays on page]
+
+**Actual behavior:**
+[ie: New Warehouse mapping disappears after refreshing the page]
+
+**Screenshots:**
+[If applicable, add screenshots to help explain your problem.]
+
+**Related logs:**
+[Please attach logs as files, and describe them here.]
```
Comment (dFed17) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: Might be good to include a link to docs on how to export logs

Comment (rickyklu) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: Some questions bug-reporting forms ask:

- Plugin version (will they know this from using the Plugin? A fancy version of a bug-submission form could  automatically populate this)
- Operating System
- Browser
- Time of day (can this automatically be generated when user submits form?)

Comment (jhoule86) [reply .github/ISSUE_TEMPLATE/bug_report.md:33]: - Thanks! Plugin version was in my original draft but it's missing here!
- 
- github will track the creation date of this, but it's not always going to be true that `time of report` is equal to `time it is experienced`
- 
- OS and browser were on the suggested template but I thought it was getting too noisy. will consider bringing them back.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+about: Report an issue with the Wayfair plugin
+title: "[New Bug]"
+labels: bug, new
+assignees: ''
+
+---
+
+**Date(s) issue was encountered**:
+**Supplier's client ID (if known):**
+**Wayfair plugin Version (displayed in `Deployed` colum in the details of a Plentymarkets Pluign Set)**
```
Comment (rickyklu) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
**Wayfair plugin Version (displayed in `Deployed` column in the details of a Plentymarkets Plugin Set)**
```

**Pull request merged**